### PR TITLE
chore(prettier): add pnpm prettignore and format

### PR DESCRIPTION
**Pull request by philibea**
run `pnpm format` 

it's was not run from a long time 🚶🏻

### CODE REVIEW

File: .prettierignore [hunk 1]
```diff
@@ -4,3 +4,4 @@ dist/
 
 # Specific to prettier
 package.json
+CHANGELOG.md
```
Comment (QuiiBz) [inline .prettierignore:7]: Any reason for not formatting this?

File: packages/use-query-params/README.md [hunk 2]
```diff
@@ -9,10 +9,11 @@ $ pnpm add @scaleway/use-query-params react-router-dom
 ```
 
 ## Usage
+
 ```js
 import React from 'react'
```
Comment (QuiiBz) [inline packages/use-query-params/README.md:14]: Is this react import mandatory?

File: .prettierignore (hunk 1 above)
Comment (philibea) [reply .prettierignore:7]: it's automatically created by release. it's will make some noise when we want to format

File: packages/use-query-params/README.md (hunk 2 above)
Comment (philibea) [reply packages/use-query-params/README.md:14]: We don't know if our user use new jsx transform ?

File: packages/countries/subdivisions/AD.json [hunk 3]
```diff
@@ -48,4 +48,4 @@
     "parent_code": null,
     "type": "Parish"
   }
-]
\ No newline at end of file
+]
```
[the remark this answers is not in the document]
Comment (philibea) [reply packages/countries/subdivisions/AD.json:51]: auto generated by us, we can still apply format ?

[the remark this answers is not in the document]
Comment (chambo-e) [reply packages/countries/subdivisions/AD.json:51]: We need to add json and d.ts files to lint-staged then :)

[the remark this answers is not in the document]
Comment (philibea) [reply packages/countries/subdivisions/AD.json:51]: Rigght, it's done

**Review by chambo-e — APPROVED** [review]

**Review by QuiiBz — APPROVED** [review]

**Pull request merged**